R/zchunk_L233.water.demand.livestock.R

```diff
@@ -34,37 +35,34 @@ module_water_L233.water.demand.livestock_DISABLED <- function(command, ...) {
 
     # ===================================================
     # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+    #Just read in water coefficients for all years
+    L133.water_demand_livestock_R_C_W_km3_Mt %>%
+      rename(supplysector = GCAM_commodity) %>%
+      inner_join(select(A_an_technology, supplysector, subsector, technology), by = "supplysector") %>%
+      mutate(water_sector = "Livestock") %>%
+      mutate(minicam.energy.input = set_water_input_name(water_sector, water_type, A03.sector)) %>%
+      left_join(GCAM_region_names, by = "GCAM_region_ID") %>%
+      mutate(market.name = region) -> L233.TechCoef_tmp
+      # Set the coef for all years
+    L233.TechCoef_tmp[rep(1:nrow(L233.TechCoef_tmp), times = length(MODEL_YEARS)), ] %>%
+      mutate(year = MODEL_YEARS[sort(rep(1:length(MODEL_YEARS), times = nrow(L233.TechCoef_tmp)))]) %>%
```

Comment:
These two lines can be replaced by the MUCH simpler
```R
      repeat_add_columns(tibble(year = MODEL_YEARS)) %>%
```

I'll make and commit this change.